src/main/java/tool/arenasizer/ArenaInfoPanel.java

```diff
@@ -0,0 +1,174 @@
+        expandedCapacityPanel.labelPercentageVipBoxes.setPercentNumber(stadium.isAusbau() ? BigDecimal.valueOf(stadium.getAusbauLogen()).divide(BigDecimal.valueOf(stadium.getLogen()), 3, RoundingMode.HALF_DOWN) : null);
+        expandedCapacityPanel.labelPercentageTotal.setPercentNumber(stadium.getAusbauGesamtgroesse().map(expansionTotal -> BigDecimal.valueOf(expansionTotal).divide(BigDecimal.valueOf(stadium.getGesamtgroesse()), 3, RoundingMode.HALF_DOWN)).orElse(null));
+
+        expandedCapacityPanel.label2.setText(Optional.ofNullable(stadium.getExpansionDate()).map(expansionDate -> expansionDate.toLocaleDateTime()).orElse(EMPTY));
+
+        // Future
+        futureCapacityPanel.labelCountTerraces.setText(stadium.getZukunftStehplaetze().map(numberformat::format).orElse(notAvailableString));
+        futureCapacityPanel.labelCountBasicSeating.setText(stadium.getZukunftSitzplaetze().map(numberformat::format).orElse(notAvailableString));
+        futureCapacityPanel.labelCountSeatsUnderRoof.setText(stadium.getZukunftUeberdachteSitzplaetze().map(numberformat::format).orElse(notAvailableString));
+        futureCapacityPanel.labelCountVipBoxes.setText(stadium.getZukunftLogen().map(numberformat::format).orElse(notAvailableString));
+        futureCapacityPanel.labelCountTotal.setText(stadium.getZukunftGesamtgroesse().map(numberformat::format).orElse(notAvailableString));
+
+        futureCapacityPanel.labelPercentageTerraces.setText(stadium.getZukunftStehplaetze().map(future -> percentString(future, stadium.getZukunftGesamtgroesse().get())).orElse(notAvailableString));
+        futureCapacityPanel.labelPercentageBasicSeating.setText(stadium.getZukunftSitzplaetze().map(future -> percentString(future, stadium.getZukunftGesamtgroesse().get())).orElse(notAvailableString));
+        futureCapacityPanel.labelPercentageSeatsUnderRoof.setText(stadium.getZukunftUeberdachteSitzplaetze().map(future -> percentString(future, stadium.getZukunftGesamtgroesse().get())).orElse(notAvailableString));
+        futureCapacityPanel.labelPercentageVipBoxes.setText(stadium.getZukunftLogen().map(future -> percentString(future, stadium.getZukunftGesamtgroesse().get())).orElse(notAvailableString));
+        futureCapacityPanel.labelPercentageTotal.setText(stadium.getZukunftGesamtgroesse().map(future -> percentString(future, stadium.getZukunftGesamtgroesse().get())).orElse(notAvailableString));
+
+        futureCapacityPanel.label2.setText(Optional.ofNullable(stadium.getExpansionDate())
+                .map(expansionDate -> nextDay(expansionDate).toLocaleDate())
+                .orElse(EMPTY));
+
+        setTranslation();
+    }
+
+    private static HODateTime nextDay(HODateTime hoDateTime) {
+        var localDate = hoDateTime.getLocalDateTime().toLocalDate();
+        var nextDay = localDate.plusDays(1);
+        var instant = Instant.from(nextDay.atStartOfDay(HODateTime.DEFAULT_TIMEZONE));
+        return new HODateTime(instant);
+    }
```

Comment:
Looks like a good method of HODateTime

Reply:
Review Finding 4: Move 'nextDay' to 'HoDateTime.nextLocalDay'